UnityProject/Assets/Scripts/Messages/Server/JoinedViewer/JobRequestFailedMessage.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections;
+using UnityEngine;
+using Mirror;
+
+namespace Messages.Server
+{
+	public class JobRequestFailedMessage : ServerMessage
+	{
+		public JobRequestError FailReason;
+
+		public override void Process()
+		{
+			UIManager.Display.jobSelectWindow.GetComponent<GUI_PlayerJobs>().ShowFailMessage(FailReason);
+		}
+
+		public static JobRequestFailedMessage SendTo(ConnectedPlayer recipient, JobRequestError failReason)
+		{
+			var msg = new JobRequestFailedMessage
+			{
+				FailReason = failReason,
+			};
+
+			msg.SendTo(recipient);
+			return msg;
+		}
+
+		public override void Deserialize(NetworkReader reader)
```

Comment:
I don't think you need these?
I think it's only needed when you doing custom serialisation

Reply:
You're right. Fixed.